python/ray/serve/tests/test_multiplex.py

```diff
@@ -0,0 +1,292 @@
+import pytest
```

Comment:
Could you add a test with an object that has an explicit `__del__` method to check that the method gets executed?